bci_tester/data.py

```diff
@@ -1115,6 +1148,7 @@ def create_BCI(
     HELM_CONTAINER,
     *COSIGN_CONTAINERS,
     *KUBECTL_CONTAINERS,
+    *KIOSK_CONTAINERS,
```

Comment:
Wrong list, they contain zypper

Reply:
well, I'm making them not  zypper based - see https://github.com/SUSE/BCI-dockerfile-generator/pull/2537